<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeLP</class>
 <widget class="QDialog" name="changeLP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Смена логина/пароля</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Новый логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newLog"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Новый пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newPass"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Старый пароль*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="oldPass"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* - Обязательные поля</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>changeLP</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>changeLP</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
